webapp/locales/en.json

```diff
@@ -31,6 +31,9 @@
         "title": "Join Human Connection!",
         "form": {
           "description": "To get started, enter your email address:",
+          "terms-and-condition": "Our <a href=\"/terms-and-conditions\"><ds-text bold color=\"primary\" > terms of conditions!</ds-text></a> apply!",
```

Comment:
I would not use exclamation marks. (!)

Reply:
That's agreed with martin. 
because you have to put a checkmark at the registration .. there is only a hint with a link to the infosource at this place. 

at this place no little checkers should be set. 

but no worries I wanted to put them also there ran. 

please talk to martin about such suggestions.